lib/tasks/links_migration/link_migrator.rb

```diff
@@ -0,0 +1,45 @@
+module Tasks
+  module LinksMigration
+    class LinkMigrator
+      def populate_topic_links
+        relevant_subscriber_lists.each do |sl|
+          base_path = base_path_from(sl)
+          content_item = Services.content_store.content_item(namespaced(base_path))
+          if content_item && content_item.content_id.present?
```

Comment:
Redirects.

Good shout though. Both methods were originally one thing, but in the current form of this class it probably makes sense to fail loudly if this is the case - the idea being the cleanup task below should be run first. Will update...